metric-schema-markdown/src/test/java/com/palantir/metric/schema/markdown/MarkdownRendererTest.java

```diff
@@ -246,6 +246,37 @@ void testComplexTagged() {
                         + "  - `processing` values (`foo`,`bar`)");
     }
 
+    @Test
+    void testTaggedWithDocsOnlyOnTag() {
+        MetricSchema schema = MetricSchema.builder()
+                .namespaces(
+                        "namespace",
+                        MetricNamespace.builder()
+                                .docs(Documentation.of("namespace docs"))
+                                .metrics(
+                                        "metric",
+                                        MetricDefinition.builder()
+                                                .type(MetricType.METER)
+                                                .tagDefinitions(TagDefinition.builder()
+                                                        .name("result")
+                                                        .docs(Documentation.of("This is a result tag"))
+                                                        .build())
+                                                .docs(Documentation.of("metric docs"))
+                                                .build())
+                                .build())
+                .build();
+        String markdown = MarkdownRenderer.render(
+                "com.palantir:test", ImmutableMap.of("com.palantir:test:1.0.0", ImmutableList.of(schema)));
+        assertThat(markdown)
+                .isEqualTo("# Metrics\n\n"
+                        + "## Test\n\n"
+                        + "`com.palantir:test`\n\n"
+                        + "### namespace\n"
+                        + "namespace docs\n"
+                        + "- `namespace.metric` (meter): metric docs\n"
+                        + "  - `result`: This is a result tag");
```

Comment:
previously this was rendered as the following, missing the `result` tag docs entirely:
> - `namespace.metric` tagged `result` (meter): metric docs